plugins/org.yakindu.sct.doc.user/src/user-guide/sctunit.textile

```diff
@@ -5,370 +5,941 @@ p.
 
-bc(prettyprint). cycle
 
-p. ==<!-- End sctunit_keyword_cycle -->==
 ==<!-- Start sctunit_keyword_enter -->==
+
+h4(#sctunit_entering_a_state_machine). Entering a state machine
+
+The _enter_ statement serves to enter the statechart associated with this "test class":#sctunit_test_classes. The state machine is initialized and started. The state that is denoted by the initial state becomes active.
+
+A test must execute the _enter_ statement before it can perform any sensible testing on the statechart. Unless the state machine is entered, all states are inactive.
+
+Please see section "&quot;SCTUnit by example&quot;":#sctunit_sctunit_by_example for examples on how the _enter_ statement is used. Please also see the "section on the _exit_ statement":#sctunit_exiting_a_state_machine.
+
+The _enter_ statement's grammar is simple, as figure "&quot;Enter statement grammar&quot;":#sctunit_fig_enter_grammar shows:
+
+p(#sctunit_fig_enter_grammar). 
+
+!images/sctunit_grammar_020_enterstatement.png(Enter statement grammar)!
+
+p=. Enter statement grammar
+
+==<!-- End sctunit_keyword_enter -->==
+
+
+
 ==<!-- Start sctunit_keyword_exit -->==
 
-h4. enter / exit
+h4(#sctunit_exiting_a_state_machine). Exiting a state machine
+
+The _exit_ statement exits and quits a state machine. You can re-initialize and re-enter it using the "_enter_ statement":#sctunit_entering_a_state_machine.
+
+
+==<div class="example">==
+
+Example:
+
+The statechart _myStatechart_ looks like this:
+
+!(tiny)images/sctunit_myStatechart_010_simple.png(Statechart _myStatechart_ )!
+
+p=. Statechart _myStatechart_
+
+The _enter_ and _exit_ statements are explained by the comments of this test class:
 
-The 'enter' statements marks the entry of a statechart and is mandatory in every test. The 'exit' statement is only used if a statechart is leaved while the test is running. 
+bc.. 
+testclass enter_exit_tests for statechart myStatechart {
 
-bc(prettyprint). enter
-exit
+    @Test
+    operation checkState () {
+
+        /* Before entering the state machine, all states are inactive: */
+        assert !active (myStatechart.main_region.State_A)
+
+        /* Now we are entering the state machine. The state that the
+         * initial state points to becomes active: */
+        enter
+        assert active (myStatechart.main_region.State_A)
+
+        /* The "exit" statement leaves the state machine. All of the 
+         * latter's states are thus inactive:
+         */
+        exit
+        assert !active (myStatechart.main_region.State_A)
+
+        /* It is possible to re-enter the state machine or to be precise:
+         * to enter a new instance of the state machine. As above,
+         * "State_A" should be active now: */
+        enter
+        assert active (myStatechart.main_region.State_A)
+    }
+}
+p. 
+
+==</div>==
+
+The _exit_ statement's grammar is simple, as figure "&quot;Exit statement grammar&quot;":#sctunit_fig_exit_grammar shows:
```

Comment:
If you view the documentation in the browser, this line is in the coding example.

Please fix this